includes/admin/home/help/class-sensei-home-help-provider.php

```diff
@@ -14,25 +14,74 @@ class Sensei_Home_Help_Provider {
 	/**
 	 * Return a list of categories which each contain multiple help items.
 	 *
-	 * @return Sensei_Home_Help_Category[]
+	 * @return array[]
 	 */
 	public function get(): array {
 		return [
-			new Sensei_Home_Help_Category(
+			$this->create_category(
```

Comment:
Since we're simplifying it, I wonder if we have a big benefit on having the `create_*` methods, or if we could just return the full array directly.

Reply:
The same question applies to the other providers. :)

Reply:
IMHO, I feel like the `create_*` methods are a good middle ground already, like, it has some value while reading the code to see that the nested structure is actually a category, or something else.

It's similar to having a data class implement `JsonSerializable` and then implement the `jsonSerialize` method to return the proper array structure. With that said, I think this approach is okay, as too much simplicity would also affect reading code easily. :)

Reply:
Hey,  thank you both.

I agree that while that would in fact simplify it even more as @fjorgemota said I still see some benefits about readability and future error prevention having them (it's easier to add a new category or item with the correct schema).

I think I am keeping them if there is no other strong opinion :)

Reply:
That's fine! Thank you for checking, folks! =)